tests/gis_tests/distapp/tests.py

```diff
@@ -224,6 +224,20 @@ def test_mysql_geodetic_distance_error(self):
         with self.assertRaisesMessage(ValueError, msg):
             AustraliaCity.objects.filter(point__distance_lte=(Point(0, 0), D(m=100))).exists()
 
+    @skipUnlessDBFeature("supports_dwithin_lookup")
+    def test_dwithin_subquery(self):
+        """
+        Test that dwithin lookups can be used in a subquery using OuterRef as a parameter.
+
+        Tests bug #30687 where queries of this form failed to compile
```

Comment:
You can drop the ticket reference as we can walk back to tickets by blaming commits and you can remove the `Tests...` suffixes as they don't add much context; we expect these methods to test something

The following should do.

```python
"""dwithin lookup can be used in a subquery using OuterRef as a parameter."""
```